src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -35,6 +38,16 @@ import { ILogService } from 'vs/platform/log/common/log';
 		</div>
 		<div [style.display]="getEditableDisplay()" #editableDropDown style="width: 100%;"></div>
 		<div [style.display]="getNotEditableDisplay()" #dropDown style="width: 100%;"></div>
+		<label #errorMessage tabindex="-1"
+		aria-live="polite" [attr.id]="errorId" aria-atomic="true"
+		*ngIf="!_valid && validationErrorMessages && validationErrorMessages.length!==0 && !isInitState">
```

Comment:
![image](https://user-images.githubusercontent.com/6816294/132769903-1329b877-1f2f-4d0c-98b2-6b766732e749.png)

Reply:
Generally speaking, there shouldn't be multiple validation messages. However, it is better to support them now and force everyone to use arrays instead of adding them later with back compat cases everywhere.